model_analyzer/record/metrics_manager.py

```diff
@@ -354,11 +359,11 @@ def _create_model_variant(
                 self._first_config_variant.setdefault(original_name, None)
 
                 if ignore_first_config_variant:
-                    variant_config.write_config_to_file(
+                    variant_config.model_config.write_config_to_file(
```

Comment:
I feel like users using `ModelConfigVariant` shouldn't need to know to access internal `model_config` object to call this method. It would be better if `ModelConfigVariant` support `write_config_to_file` functionality directly.

Reply:
It's a slippery slope bringing up methods from the child class. Obviously, we do bring up some methods as high as the ModelRunConfig class, but I think it's better to keep the MCV class as a pure dataclass.